graas/hardware.py

```diff
@@ -0,0 +1,85 @@
+from twisted.internet import task, reactor
+from twisted.internet.defer import succeed, inlineCallbacks, returnValue
+
+
+class AsyncIO(object):
+
+    def __init__(self, GPIO):
+        '''
+        GPIO - Raspberry Pi GPIO module to control hardware
+        '''
+        GPIO.setmode(GPIO.BOARD)
+        self.pin_state = {}
+
+        self.INPUT = GPIO.IN
+        self.OUTPUT = GPIO.OUT
+        self.HIGH = GPIO.HIGH
+        self.LOW = GPIO.LOW
+        self.PULL_UP = GPIO.PUD_UP
+        self.PULL_DOWN = GPIO.PUD_DOWN
```

Comment:
I originally had them on the class, but I wanted to be able to pass in the `GPIO` class on initialization so that I could pass in a fake class for testing.